Add new fix method to set variable with element value #282 (#365)
metafacture/metafacture-fix: README.md

[Java Code](https://github.com/search?type=code&q=repo:metafacture/metafacture-fix+path:FixMethod.java+"+put_vars+{")

##### `to_var`

> Defines a single global variable that can be referenced with `$[<variableName>]` and assinges the value of the `<sourceField>`.

Author: ```suggestion
Defines a single global variable that can be referenced with `$[<variableName>]` and assigns the value of the `<sourceField>`.
```

> Defines a single global variable that can be referenced with `$[<variableName>]` and assinges the value of the `<sourceField>`.

Author: Done. Thanks.

```perl

> to_var("<sourceField>", "<variableValue>")

Author: ```suggestion
to_var("<sourceField>", "<variableName>")
```

> to_var("<sourceField>", "<variableValue>")

Author: Done. Thanks.

```

Options:

> - `"default"`: provide default value if source field does not exist.

Reviewer: Drop quotes, capitalize first word after colon.

Option description should say what it _is_, not what it _does_.

```suggestion
- `default`: Default value if source field does not exist.
```

Author: Isn't it important to highlight that this option has to be in quotes?

Reviewer: See [discussion in #354](https://github.com/metafacture/metafacture-fix/pull/356#discussion_r1799506405).

Author: ```suggestion
- `default`: Default value if source field does not exist. The option needs to be written in quotation marks because it is a reserved word in Java. (Default: Empty string)
```

> 

Reviewer: Extra empty line.

Author: ```suggestion
```
